docs/designs/data_transform.md

```diff
@@ -158,9 +158,7 @@ We can't use SQL directly to do the same analysis work as above.
 
 #### Proposal
 
-We can provide common tools to normalize the table schema. If the data is stored in Odps table, we can use [PyOdps](https://pyodps.readthedocs.io/en/latest/) + [UDF](https://www.alibabacloud.com/help/doc-detail/73359.htm) to complete the task.  
-
-*Please check the [discussion](https://github.com/sql-machine-learning/elasticdl/issues/1648).*
+We can provide common tools to normalize the table schema. If the data is stored in Odps table, we can use [PyOdps](https://pyodps.readthedocs.io/en/latest/) + [UDF](https://www.alibabacloud.com/help/doc-detail/73359.htm) to complete the task. We can look at the doc [flatten_odps_key_value_table](./flatten_odps_key_value_table.md) for the detailed design.  
```

Comment:
"Odps table" ->  "MaxCompute table".  "ODPS" it the old name of "MaxCompute".
"PyOdps" -> "PyODPS"

Reply:
Good catch, updated.